satellite/satellitedb/overlaycache.go

```diff
@@ -410,6 +410,38 @@ func (cache *overlaycache) Reliable(ctx context.Context, criteria *overlay.NodeC
 	return nodes, nil
 }
 
+// ReliableWithAddress returns map all reliable nodes to their addresses.
+func (cache *overlaycache) ReliableWithAddress(ctx context.Context, criteria *overlay.NodeCriteria) (_ map[storj.NodeID]string, err error) {
+	// get reliable and online nodes
+	query := cache.db.Rebind(`
```

Comment:
Any reason we cannot do this query with dbx?

Reply:
Did it the same way as the query above

Reply:
I'm not sure why that one is written that way either.